build/transpile.js

```diff
@@ -441,6 +441,8 @@ class Transpiler {
             [ /typeof\s+([^\s]+)\s+\!\=\=?\s+\'number\'/g, "!(is_float($1) || is_int($1))" ],
 
             [ /undefined/g, 'null' ],
+            [ /\} else if/g, '} elseif' ],
+            [ /else if/g, 'elseif' ],
```

Comment:
is this supposed to be in this PR?

Reply:
it's a minor edit that's not related but i've snuck it into here